pallets/transfer-allowlist/src/benchmarking.rs

```diff
@@ -0,0 +1,298 @@
+			verify{
+					assert_eq!(
+
+							  Pallet::<T>::get_account_currency_restriction_count_delay(
+									  sender,
+									  CurrencyId::Native,
+							  ).unwrap(),
+					  AllowanceMetadata {
+							  allowance_count: 1,
+							  current_delay: None,
+							  once_modifiable_after: None
+					  }
+
+					)
+			}
+
+
+  remove_transfer_allowance_delay_present {
+		  let (sender, receiver) = set_up_users::<T>();
+	  let delay = T::BlockNumber::one();
+		  Pallet::<T>::add_allowance_delay(RawOrigin::Signed(sender.clone()).into(), CurrencyId::Native, delay.clone())?;
+			Pallet::<T>::add_transfer_allowance(RawOrigin::Signed(sender.clone()).into(), CurrencyId::Native, receiver.clone().into())?;
+		  advance_n_blocks::<T>(1u32.into());
+  }:remove_transfer_allowance(RawOrigin::Signed(sender.clone()), CurrencyId::Native, receiver.clone().into())
+			verify {
+				  assert_eq!(
+						  Pallet::<T>::get_account_currency_transfer_allowance(
+								  (sender,
+								   CurrencyId::Native,
+								   Location::from(receiver))
+						  ).unwrap(),
+							AllowanceDetails {
+								  allowed_at: T::BlockNumber::one(),
+								  blocked_at: <frame_system::Pallet<T>>::block_number().checked_add(&delay).expect("Invalid blocked at."),
+							}
+					)
+			}
+
+	remove_transfer_allowance_no_delay {
+		  let (sender, receiver) = set_up_users::<T>();
+			Pallet::<T>::add_transfer_allowance(RawOrigin::Signed(sender.clone()).into(), CurrencyId::Native, receiver.clone().into())?;
+	}:remove_transfer_allowance(RawOrigin::Signed(sender.clone()), CurrencyId::Native, receiver.clone().into())
+		  verify {
+			  assert_eq!(
+					  Pallet::<T>::get_account_currency_transfer_allowance(
+							  (sender,
+							   CurrencyId::Native,
+							   Location::from(receiver))
+					  ).unwrap(),
+						AllowanceDetails {
+							  allowed_at: T::BlockNumber::zero(),
+							  blocked_at: <frame_system::Pallet<T>>::block_number(),
+						}
+				)
+		  }
+
+	purge_transfer_allowance_no_remaining_metadata {
+			let (sender, receiver) = set_up_users::<T>();
+			  Pallet::<T>::add_transfer_allowance(RawOrigin::Signed(sender.clone()).into(), CurrencyId::Native, receiver.clone().into())?;
+			Pallet::<T>::remove_transfer_allowance(RawOrigin::Signed(sender.clone()).into(), CurrencyId::Native, receiver.clone().into())?;
+	}:purge_transfer_allowance(RawOrigin::Signed(sender.clone()), CurrencyId::Native, receiver.clone().into())
+	verify {
+			  assert_eq!(
+					  Pallet::<T>::get_account_currency_transfer_allowance(
+							  (sender.clone(),
+							   CurrencyId::Native,
+							   Location::from(receiver))
+					  ),
+			None
+				);
+			assert_eq!(
+
+					Pallet::<T>::get_account_currency_restriction_count_delay(
+							sender,
+							CurrencyId::Native,
+					),
+					None
+			)
+	}
+
+	  purge_transfer_allowance_remaining_metadata {
+			  let (sender, receiver) = set_up_users::<T>();
+		let receiver_1 = set_up_second_reciever::<T>();
```

Comment:
All benchmarks are not alignment due to be in the `benchmarks!` macro. We need to check it "manual". It's annoying, I know...